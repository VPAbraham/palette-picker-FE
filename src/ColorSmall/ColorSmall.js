import React from 'react';
import './ColorSmall.scss';

export const ColorSmall = (props) => {
  const style = {
    background: props.color
  }
  return (
    <div style={style} id="color-small">
    </div>
  )
}

export default ColorSmall;
