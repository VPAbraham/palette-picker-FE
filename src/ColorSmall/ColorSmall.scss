#color-small {
  height: 3vh;
  width: 3vh;
  border-radius: 1px;
}
