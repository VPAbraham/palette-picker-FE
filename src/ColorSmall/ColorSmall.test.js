import React from 'react';
import { ColorSmall } from './ColorSmall';
import { shallow } from 'enzyme';

describe('ColorSmall', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ColorSmall
      color={'#baddc3'}
      key='3'
    />)
  });
  it('should match the snapshot with all data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});