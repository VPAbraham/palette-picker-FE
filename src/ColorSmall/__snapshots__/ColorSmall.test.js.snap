// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorSmall should match the snapshot with all data passed in 1`] = `
<div
  id="color-small"
  style={
    Object {
      "background": "#baddc3",
    }
  }
/>
`;
